docs/commands/preview-docs.md

```diff
@@ -1,39 +1,98 @@
-  --port, -p               Preview port.                [number] [default: 8080]
-  --skip-preprocessor      Ignore certain preprocessors.                 [array]
-  --skip-decorator         Ignore certain decorators.                    [array]
-  --use-community-edition  Force using Redoc CE for docs preview.      [boolean]
-  --force, -f              Produce bundle output even when errors occur.
-                                                                       [boolean]
-  --config                 Specify path to the config file.             [string]
+```bash
+openapi preview-docs openapi/openapi.yaml
 ```
 
+In this case, `preview-docs` will preview the definition that was passed to the command. The configuration file is ignored.
 
-### How to preview the docs on a custom port
+#### Pass entrypoint without extension
 
-By default, without providing a port, the preview starts on port 8080, and can be accessed at `http://localhost:8080`.
+You can omit entrypoint's file extension when executing the `preview-docs` command. In this way, you can reference either `.yaml` or `.json` file.
 
-This command starts a preview on port 8888, and you can access the docs at `http://localhost:8888` after running it.
+```bash
+# preview-docs will preview either petstore.yaml or petstore.json file in the current working directory
+openapi preview-docs petstore
+# preview-docs will preview either sandbox.yaml or sandbox.json file in the openapi/extra directory
+openapi preview-docs openapi/extra/sandbox
+```
+
+#### Pass entrypoint via configuration file
+
+Instead of a full path, you can use an alias assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints. For example, `petstore`:
 
+```bash command
+openapi preview-docs petstore
+```
+
+```yaml .redocly.yaml
+apiDefinitions:
+  petstore: ./openapi/petstore-definition.json
+```
 
-```shell
+In this case, after resolving the path behind the `petstore` alias (see the `.redocly.yaml` tab), `preview-docs` will preview the `petstore.json` definition file. The presence of the `.redocly.yaml` configuration file is mandatory.
+
+### Custom configuration file
+
+By default, the CLI tool looks for a `.redocly.yaml` configuration file in the current working directory. Use the optional `--config` argument to provide an alternative path to a configuration file.
+
+```bash
+openapi preview-docs --config=./another/directory/config.yaml
+```
+
+### Custom port for preview
+
+By default, without providing the port option, the preview starts on port `8080`, so you can access the docs at `http://localhost:8080`
+
+To specify a custom port for the preview, pass the desired value using either short or long option format:
+
+```bash short format
 openapi preview-docs -p 8888 openapi/openapi.yaml
 ```
+
+```bash long format
+openapi preview-docs -port 8888 openapi/openapi.yaml
+```
+
+Both commands will start the preview on port `8888`, so you can access the docs at `http://localhost:8888`
```

Comment:
```suggestion
Both commands will start the preview on port `8888`, so you can access the docs at `http://localhost:8888`.
```

Reply:
Full stop after links is considered bad practice because people can unintentionally copy-paste that symbol and get confused by unexpected results.

Reply:
Right, good point... I just wanted to end the sentence :sweat_smile: 
You can dismiss this suggestion, then.